Lib/importlib/metadata.py

```diff
@@ -7,15 +7,18 @@
 import pathlib
 import zipfile
 import operator
+import textwrap
 import warnings
 import functools
 import itertools
 import posixpath
+import contextlib
```

Comment:
`suppress` is already imported at https://github.com/python/cpython/pull/25297/files#diff-319dbaf16f78143c0043addc0e89ee8c44594c861da9da3c160f3598d2625a35R22

Reply:
Fixed in https://github.com/python/importlib_metadata/commit/5595f2dc2a2ab4b7d1c7cf2488c427cfc134f184, which will be released in a subsequent `importlib_metadata` release and later merged into CPython. If it's okay with you, I'll leave the irregular code here for now.

Reply:
Turns out I needed to sync other things too, so fixed in 3063cf31d2.